NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh

```diff
@@ -1,16 +1,8 @@
 #!/bin/bash
 set -x
 basedir=$( cd "$(dirname "$0")" ; pwd -P )
-conda env update -n base --solver=libmamba -f ${basedir}/env_main_ADE.yaml
+# Install dependencies
+conda env update -n r -f ${basedir}/env_main_ADE.yaml
 
-# conda install conda-forge::r-fmesher --yes
-# conda install conda-forge::r-exactextractr --yes
-# conda install conda-forge::r-sn --yes
-# conda install conda-forge::r-MatrixModels --yes
-# conda install conda-forge::r-inlabru --yes
-# conda install conda-forge::r-terra --yes
-# conda install conda-forge::r-dplyr --yes
-# conda install conda-forge::r-spdep --yes
-# conda install conda-forge::r-sf --yes
-# conda install conda-forge::r-fields --yes
-# conda install conda-forge::r-Matrix --yes
\ No newline at end of file
+# Install INLA
+conda run --name r --no-capture-output Rscript ${basedir}/install.R
```

Comment:
Where is `install.R`? (Also, add a newline to the end of this file)

Reply:
Added now